<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelUserName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRePassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUserName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRePassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUserType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxUserType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>客户</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>管理员</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelCompanyName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航司：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCompanyName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
